charts/prometheus-server/Chart.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: prometheus-server
+appVersion: 2.39.1
+version: 1.0.0
+description: Prometheus is a monitoring system and time series database.
+home: https://prometheus.io/
+icon: https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/assets/prometheus_logo-cb55bb5c346.png
+sources:
+  - https://github.com/prometheus/prometheus-server
+maintainers:
+  - name: ivyostosh
+    email: ivyjin215@gmail.com
```

More than single maintainer would be better.

```suggestion
    email: ivyjin215@gmail.com
  - name: MH
    email: zanhsieh@gmail.com
```

---

Updated